fedn/cli/shared.py

```diff
@@ -64,26 +64,36 @@ def get_client_package_dir(path: str) -> str:
 
 
 # Print response from api (list of entities)
-def print_response(response, entity_name: str):
+def print_response(response, entity_name: str, so):
     """Prints the api response to the cli.
     :param response:
         type: array
         description: list of entities
     :param entity_name:
         type: string
         description: name of entity
+    :param so:
+        type: boolean
+        desriptions: single output format (y/n)
     return: None
     """
     if response.status_code == 200:
         json_data = response.json()
-        count, result = json_data.values()
-        click.echo(f"Found {count} {entity_name}")
-        click.echo("\n---------------------------------\n")
-        for obj in result:
-            click.echo("{")
-            for k, v in obj.items():
+        if so:
+            click.echo(f"Found {entity_name}")
+            click.echo("\n---------------------------------\n")
+            for k, v in json_data.items():
                 click.echo(f"\t{k}: {v}")
-            click.echo("}")
+        else:
+            count, result = json_data.values()
```

Comment:
we should consider using json pretty prints instead, however I would prefer not introducing a new dependency. Save for other issue